docs/manual/exporting-the-schedule/index.md

```diff
@@ -252,15 +252,15 @@ If you have secure shell access (SSH) to the remote web server, you could write
 Website widgets
 ---------------
 
-Example HTML, Javascript and CSS code for your public website are provided in the *widgets* folder of the Airtime installation tarball, or on GitHub: <https://github.com/sourcefabric/Airtime/tree/master/widgets>
+Example HTML, Javascript and CSS code for your public website are provided in the *widgets* folder of the Libretime installation tarball, or on GitHub: <https://github.com/sourcefabric/Libretime/tree/master/widgets>
 
-If you have performed an automated installation on Debian or Ubuntu, the widgets can be found in the */usr/share/doc/airtime/examples/* directory.
+If you have performed an automated installation on Debian or Ubuntu, the widgets can be found in the */usr/share/doc/libretime/examples/* directory.
```

Comment:
This is not valid (related to no example widgets)

Reply:
These are in the 2.5 and 2.6 branches, are they not? 
Should I remove references to the example widgets until/unless they are merged into current code?

Reply:
Yeah I'd just remove them, there is a widget page on the LibreTime install that I think was added after 2.5 and the widgets that exist are mentioned above.

Reply:
Done.